docs/source/accelerators/hpu.rst

```diff
@@ -0,0 +1,212 @@
+.. _hpu:
+
+Habana Gaudi AI Processor (HPU)
+===============================
+
+Habana® Gaudi® AI training processors have been architected from the ground up and optimized for deep learning training efficiency.
+Gaudi offers substantial price/performance advantage -- so you get to do more deep learning training while spending less.
+
+You can use either `the Gaudi-based AWS EC2 DL1 instances <https://aws.amazon.com/ec2/instance-types/dl1/>`_ or `the Supermicro X12 Gaudi server <https://www.supermicro.com/en/solutions/habana-gaudi>`_.
+
+Habana’s SynapseAI® software suite is optimized for building and training deep learning models using TensorFlow and PyTorch frameworks. Gaudi is referred to as the Habana Processing Unit (HPU).
+With SynapseAI, we aim to make training workloads on Gaudi easy, whether you're developing from scratch or migrating existing workloads.
+
+For more information, check out `<https://developer.habana.ai>`_ and `<https://habana.ai/>`_.
+
+----------------
+
+PyTorch Lightning With Gaudi HPU
+--------------------------------
+
+Lightning supports training on a single HPU device or 8 HPU devices with the plugins described in the following sections
+
+
+----------------
+
+.. _hpu_accelerator:
+
+HPU accelerator
+---------------
+
+To enable PyTorch Lightning to utilize the HPU accelerator, simply provide ``Trainer(accelerator="hpu")`` parameters in the trainer class.
+
+
+----------------
+
+.. _single_device_strategy:
+
+Training on Single HPU
+----------------------
+
+The ``devices=1`` and ``accelerator="hpu"`` with ``strategy=SingleHPUStrategy(device=torch.device("hpu"))`` parameter in the trainer class enables the Habana backend for single Gaudi training.
```

Comment:
this is confusing. 

To enable this, do I do the following:
```py
Trainer(accelerator="hpu", devices=1)
```
? 
why do I have to specify a SingleHPUStrategy ?

Reply:
automatic selection need not have the strategy defined. Code specific to HPU initialization is kept in this strategy as per previous requests